Framework/DataObjects/src/PeaksWorkspace.cpp

```diff
@@ -149,6 +150,28 @@ void PeaksWorkspace::removePeak(const int peakNum) {
   peaks.erase(peaks.begin() + peakNum);
 }
 
+/** Removes multiple peaks
+* @param badPeaks peaks to be removed. Takes ownership of container.
+*/
+void PeaksWorkspace::removePeaks(std::vector<int> &&badPeaks) {
+  if (badPeaks.size() == 0)
```

Comment:
It is generally better practice and slightly more efficient to use `badPeaks.empty()` rather than checking `size() == 0`.